src/pages/_app.js

```diff
@@ -1,13 +1,56 @@
 import App, { Container } from 'next/app'
 import React from 'react'
 
+import Cookie from 'js-cookie'
+
+// Temporary fix to fetch dependency on 'fetches' lib
```

Comment:
We need to update the fetches lib in order to remove this fix from the boilerplate, did you spoke with @dleitee ?

Reply:
I removed this code because I tested here without this temporary fix and I got no more errors